parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -195,27 +181,8 @@ write_files:
   permissions: "0644"
   owner: root
   content: |
-    subreaper = false
-    oom_score = 0
-    [plugins.cri]
-    sandbox_image = "{{GetPodInfraContainerSpec}}"
-    [plugins.cri.containerd.untrusted_workload_runtime]
-    runtime_type = "io.containerd.runtime.v1.linux"
-    {{if IsKataContainerRuntime }}
```

Comment:
I'm not so familiar with this but I don't see any overrides for "Kata..." is this not used anymore?